overrides/macos-override.json

```diff
@@ -525,6 +525,10 @@
             "settings": {
                 "defaultPolicy": "brand",
                 "defaultSites": [
+                    {
+                        "domain": "chase.com",
+                        "reason": "https://github.com/duckduckgo/privacy-configuration/issues/988"
```

Comment:
Please drop a comment there about UA breakage - it looks like the original issue had a different root cause.